bin/reload-prod

```diff
@@ -6,4 +6,4 @@ git reset --hard
 git switch main
 git pull origin main
 docker compose build api frontend
-docker compose -f docker-compose.yaml -f docker-compose.prod.yaml up --no-deps -d api frontend
+docker compose -f docker-compose.yaml -f docker-compose.prod.yaml up --no-deps -d api frontend migration
```

Comment:
This ensures that we run the migration container every time we reload prod, so we run whatever migrations need running.